application/testing/CMakeLists.txt

```diff
@@ -548,6 +551,11 @@ if(VTK_VERSION VERSION_GREATER_EQUAL 9.2.20221220)
   endif()
 endif()
 
+# SSAA with HDR framebuffer support in https://gitlab.kitware.com/vtk/vtk/-/merge_requests/12026
+if(VTK_VERSION VERSION_GREATER_EQUAL 9.4.20250329)
+  f3d_test(NAME TestHDRIToneMappingSSAA DATA suzanne.ply HDRI palermo_park_1k.hdr TONE_MAPPING ARGS -t -a --anti-aliasing-mode=ssaa)
```

Comment:
I cant find a test simple anti aliasing in master, although I'm sure its tested trough default config tests.
If it indeed not exist, it may be worth adding one for fxaa.

Reply:
SSIM cannot detect such anti-aliasing algorithm